### DISCUSSION TITLE
Error log from shared subscription

### DISCUSSION THREAD

**Discussion by on-delete** [emqx/emqx#10712] [Q&A]
Currently our EMQX in version 4.4.15 is printing a lot of these error messages:

`{"time":1684219184482040,"pid":"<0.16075.192>","peername":"10.70.1.20:40086","msg":"[Channel] Unexpected info: {#Ref<0.2360806684.3192389634.64855>,shared_sub_ack}","line":1092,"level":"error","gl":"<0.1629.0>","mfa":"emqx_channel:handle_info/2"}`

Can someone explain to me, what could be the problem here? It comes with a lot of messages that are republished across other cluster nodes. General message transmission seems to be not affected.

**Comment by qzhuyan** [comment]
sounds like an uncovered use case, could you fire an issue report with debug logs? thanks!
